index.js

```diff
@@ -163,7 +163,7 @@ module.exports = {
       comments: 100,
       ignoreUrls: true,
       ignoreRegExpLiterals: true,
-      ignorePattern: '^\\s*(.+=\\s*(require|rewire)|test)\\s*\\(',
+      ignorePattern: '^\\s*((.+=\\s*(require|rewire)\\s*\\()|(\\*\\s*@[^\\s]*\\s*))'
```

Comment:
FWIW newer versions of ESLint support using arrays here for multiple patterns. Would make this more readable as two separate patterns instead of one potentially convoluted one. (see: https://github.com/eslint/eslint/pull/4123)